app/src/main/java/com/example/kotlinplannerapp/login/LoginActivityViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.kotlinplannerapp.login
+
+import android.util.Patterns
+import android.widget.Toast
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class LoginActivityViewModel : ViewModel() {
+
+    private var isLoginSuccessLiveData : MutableLiveData<Boolean> = MutableLiveData()
+
+    fun getIsLoginSuccessLiveData() : LiveData<Boolean> = isLoginSuccessLiveData
+
+    fun login(email : String, password : String) {
+        if (email.isNotEmpty() && password.isNotEmpty() && Patterns.EMAIL_ADDRESS.matcher(email).matches()) {
+            isLoginSuccessLiveData = MutableLiveData(true)
```

Comment:
[CDC] Aqui y en el else estas reasignando el valor de isLoginSuccessLiveData, poner como varibale immutable y hacer un postValue(): 
```suggestion
            isLoginSuccessLiveData.postValue(true)
```